samples/peripheral/802154_phy_test/README.rst

```diff
@@ -0,0 +1,357 @@
+     - Starts modulated waveform transmission
+     - custom lstream 100 200 20000
+   * - Get the DUT hardware version
+     - custom rhardwareversion
+     - 
+     - Gets the DUT hardware version
+     - custom rhardwareversion
+   * - Get the DUT software version
+     - custom rsoftwareversion
+     - 
+     - Gets the DUT software version
+     - custom rsoftwareversion
+   * - High frequency clock output on a selected pin
+     - custom lclk <pin> <value>
+     - <pin> - GPIO pin number, 0-N, where N depends on used SoC; <value> - enabled when 1, disabled when 0
+     - CMD will disable/enable high frequency clock output on a selected pin. Actual clock frequency depends on SoC and is the highest as it’s possible, taking in account GPIO and CLOCK modules possibilities
+     - custom lclk 10 1
+   * - Set GPIO pin value
+     - custom lsetgpio <pin> <value>
+     - <pin> - GPIO pin number, 0-N, where N depends on used SoC; <value> - high when 1, low when 0
+     - CMD will set value of the selected GPIO pin if it’s an out pin
+     - custom lsetgpio 29 0
+   * - Get GPIO pin value
+     - custom lgetgpio <pin>
+     - <pin> - GPIO pin number, 0-N, where N depends on used SoC
+     - CMD will reconfigure selected GPIO pin to INPUT mode and read its value
+     - custom lgetgpio 29
+   * - Set DC/DC mode
+     - custom lsetdcdc <value>
+     - <value> - enabled when 1, disabled when 0
+     - CMD will disable/enable DC/DC mode (no effect on unsupported boards)
+     - custom lsetdcdc 1
+   * - Get DC/DC mode
+     - custom lgetdcdc
+     - 
+     - Gets the DC/DC mode of CMD device, always 0 for unsupported boards
+     - custom lgetdcdc
+   * - Set ICACHE configuration
+     - custom lseticache <value>
+     - <value> - enabled when 1, disabled when 0
+     - CMD will disable/enable ICACHE
+     - custom lseticache 1
+   * - Read SoC temperature
+     - custom lgettemp
+     - 
+     - CMD will print SoC temperature in the following format: <.%02>
+     - custom lgettemp
+   * - Transition radio to sleep mode
+     - custom lsleep
+     - 
+     - CMD will put radio into sleep mode
+     - custom lsleep
+   * - Transition radio to receive mode
+     - custom lreceive
+     - 
+     - CMD will put radio in receive mode
+     - custom lreceive
+   * - Reboot the device
+     - custom lreboot
+     - 
+     - Reboots the device
+     - custom lreboot
+
+
+   
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/peripheral/802154_phy_test`
+
+.. include:: /includes/build_and_run.txt
+
+.. note::
+   On the |nRF5340DKnoref|, the IEEE 802.15.4 PHY Test Tool is a standalone network sample that does not require any counterpart application sample.
+   However, you must still program the application core to boot up the network core.
+   You can use any sample for this, for example :ref:`nrf5340_empty_app_core`.
```

Comment:
I'm not sure if you can use __any__ sample. It is required that app core program enables net core power, and that app core forwards UART pins to net core.